nodeup/pkg/model/logrotate.go

```diff
@@ -47,6 +47,7 @@ func (b *LogrotateBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 	}
 
 	b.addLogRotate(c, "docker", "/var/log/docker.log", logRotateOptions{})
+	b.addLogRotate(c, "kops-controller", "/var/log/kops-controller.log", logRotateOptions{})
```

Comment:
I think kops-controller is "just a pod" and doesn't write logs here.  But it is relatively important, so I think we could make the case for doing so...